public/app/features/apiserver/client.ts

```diff
@@ -90,3 +97,12 @@ export class ScopedResourceClient<T = object, K = string> implements ResourceCli
       .join(',');
   }
 }
+
+// add the origin annotations so we know what was set from the UI
+function setOriginAsUI(meta: Partial<ObjectMeta>) {
+  if (!meta.annotations) {
+    meta.annotations = {}
+  }
+  meta.annotations.AnnoKeyOriginName = "UI"
```

Comment:
would it be useful to have the grafana UI version and commit info here for debugging purposes?

Reply:
good idea!